application_test.go

```diff
@@ -198,14 +198,33 @@ func TestApplicationEnvs(t *testing.T) {
 	app := NewDockerApplication()
 	assert.Nil(t, app.Env)
 
-	app.AddEnv("hello", "world").AddEnv("foo", "bar")
-	assert.Equal(t, 2, len(*app.Env))
+	app.AddEnv("hello", "world").AddEnv("foo", "bar").AddSecret("top", "secret1", "/path/to/secret")
+	assert.Equal(t, 2, len((*app.Env)))
+	assert.Equal(t, 1, len((*app.Secrets)))
```

Comment:
I think the secrets-related tests/assertions should only happen in `TestApplicationSecret`. WDYT?